### DISCUSSION TITLE
关于如何自建随机图api的服务~

### DISCUSSION THREAD

**Discussion by huanghaonan** [LIlGG/halo-theme-sakura#404] [Q&A]
博主~如何自建随机图api的服务~

**Comment by LIlGG** [comment]
目前随机图 API 开源代码还未完全完成，可以关注 https://github.com/LIlGG/sakura-apis 仓库。
另外也有可能将在 2.0 版本中使用插件完成随机图功能，这个待定。

**Comment by huanghaonan** [comment]
> 

膜拜~
